doc-content/enterprise-only/admin-and-config/manage-tasks-con.adoc

```diff
@@ -15,8 +15,8 @@ image::admin-and-config/task-details.png[Task details]
 
 * *Work*: Displays basic details about the task and the task owner. You can click the *Claim* button to claim the task. To undo the claim process, click the *Release* button.
 * *Details*: Displays information such as task description, status, and due date.
-* *Assignments*: Displays the current owner of the task and allows you to delegate the task to another person or group.
-* *Comments*: Displays comments added by task user(s). It allows you to delete an existing comment and add a new comment.
+* *Assignments*: Displays the current owner of the task and enables you to delegate the task to another person or group.
+* *Comments*: Displays comments added by task user(s). You can delete an existing comment and add a new comment.
```

Comment:
We cannot use noun(s) to pluralize (according to the IBM style guide(page 57).

Reply:
Can you clarify what should be changed?